src/jitlayers.cpp

```diff
@@ -720,7 +720,7 @@ JuliaOJIT::JuliaOJIT(TargetMachine &TM, LLVMContext *LLVMCtx)
         static void *atomic_hdl = jl_load_dynamic_library(libatomic, JL_RTLD_LOCAL, 0);
         if (atomic_hdl != NULL) {
             GlobalJD.addGenerator(
-              std::move(*orc::DynamicLibrarySearchGenerator::Load(
+              cantFail(orc::DynamicLibrarySearchGenerator::Load(
```

Comment:
@vtjnash does this look correct to you?

Reply:
Yes, that's more correct. Thought I'd gotten this one too earlier in the Orc v2 PR.